src/appBootstrapper.jsx

```diff
@@ -43,6 +44,9 @@ initializeConfiguration();
 // THIS TELLS WEBPACK BUNDLE LOADER WHERE TO LOAD SPLIT BUNDLES
 __webpack_public_path__ = AppConfig.frontendUrl;
 
+// this fixes a chrome bug related to touch and wheel events
```

Comment:
Could you add a reference to the issue? So later we can clean it up if it's resolved?

Reply:
Sure!